crates/snapbox/src/data.rs

```diff
@@ -110,10 +107,10 @@ impl Data {
     /// Return the underlying `str`
     ///
     /// Note: this will not inspect binary data for being a valid `str`.
-    pub fn as_str(&self) -> Option<&str> {
+    pub fn try_string(&self) -> Option<String> {
```

Comment:
`try_string` doesn't make the intent clear for structured data.  What about `render` or something?

Reply:
I switched to `render` since it makes the most sense and describes what it is for